mantis-server/mantis-server-agent/src/main/java/io/mantisrx/server/agent/metrics/cgroups/CpuAcctsSubsystemProcess.java

```diff
@@ -61,4 +70,20 @@ public void getUsage(Usage.UsageBuilder resourceUsageBuilder) throws IOException
             log.warn("quota={} & period={} are not configured correctly", quota, period);
         }
     }
+
+    private void handleV2(Usage.UsageBuilder resourceUsageBuilder) throws IOException {
+        Map<String, Long> cpuStats = cgroup.getStats(null, "cpu.stat");
```

Comment:
The nullable subsystem addition here in this PR seems a little weird to me. 
I would be ok if you skipped that and just did `""` as the subsystem?

Reply:
Done.